src/index.js

```diff
@@ -1,17 +1,15 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-import { BrowserRouter } from 'react-router-dom';
+import { BrowserRouter } from "react-router-dom";
 import "./index.scss";
-import { AppRoutes } from "./AppRoutes"
+import { AppRoutes } from "./AppRoutes";
 import reportWebVitals from "./reportWebVitals";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
```

Comment:
Hey we shouldn't delete this, it's part of the application that was created in the beginning and it looks like it's a useful tool to continue using (https://reactjs.org/docs/strict-mode.html). Can you add that back in?

Reply:
Added a ticket to make this dynamic based on environments. This is what causes the double API calls on page load.

Reply:
Gonna resolve this convo because I left the next comment for this conversation on the TODO line of the code below!